### PR TITLE
Expose the primitive imports for users who really want this.

### PR DESCRIPTION
**Pull request by abizjak**
## Purpose

There have been some requests to expose the primitive imports by users who wish to squeeze out all the performance they can.

## Changes

Make primitive host functions public.

## Checklist

- [x] My code follows the style of this project.
- [x] The code compiles without warnings.
- [x] I have performed a self-review of the changes.
- [x] I have documented my code, in particular the intent of the
      hard-to-understand areas.
- [x] (If necessary) I have updated the CHANGELOG.

### CODE REVIEW

File: concordium-std/src/prims.rs [hunk 1]
```diff
@@ -1,5 +1,13 @@
 //! This module provides the primitive interface to the chain.
 //! Functions here should be wrapped in safer wrappers when used from contracts.
+//! **This module is provided for expert users who wish to optimize their smart
+//! contract to the utmost for space and size, and should not be used by the
+//! majority of users.**
+//!
+//! The functions in this module are inherently unsafe, and if preconditions
+//! that they state are not ensured well-defined, but unspecified (in this
+//! module) behaviour will occur, including memory corruption and program
+//! termination.
```
Comment (Bargsteen) [inline concordium-std/src/prims.rs:10]: This sentence is hard to read.

**Review by Bargsteen — DISMISSED** [review]
LGTM :+1:

File: concordium-std/src/prims.rs (hunk 1 above)
Comment (abizjak) [reply concordium-std/src/prims.rs]: Attempted to improve.

Comment (Bargsteen) [reply concordium-std/src/prims.rs]: Much better ;) :+1:

**Review by Bargsteen — APPROVED** [review]

**Pull request merged**